packages/container/lib/container.js

```diff
@@ -38,6 +38,7 @@ export default function Container(registry, options) {
   this.owner           = options && options.owner ? options.owner : null;
   this.cache           = dictionary(options && options.cache ? options.cache : null);
   this.factoryCache    = dictionary(options && options.factoryCache ? options.factoryCache : null);
+  this.factoryManagerCache = dictionary(options && options.factoryCache ? options.factoryCache : null);
```

Comment:
This should not share the same cache as `this.factoryCache`.  I suspect this should be:

```js
this.factoryManagerCache = dictionary(options && options.factoryManagerCache ? options.factoryManagerCache : null);
```